feat(changelog): transactional-email-changed-webhook-event-update-to-ema 2024-11-05 (#3947)
scaleway/docs-content: changelog/november2024/2024-11-05-transactional-email-changed-webhook-event-update-to-ema.mdx

---

> title: Webhook Event - Update to Email Error Field

Reviewer: ```suggestion
title: Webhook event - update to email error field
```

status: changed
author:
    fullname: 'Join the #transactional-email channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-11-05
category: managed-services
product: transactional-email
---

> The email_error field has been replaced by two new fields: response_code and response_message. This change allows for more precise error identification and improved tracking of response details. link -> https://www.scaleway.com/en/docs/managed-services/transactional-email/api-cli/webhook-events-payloads/

Reviewer: ```suggestion
The `email_error` field has been replaced by two new fields: `response_code` and `response_message`. This change allows for more precise error identification and improved tracking of response details. See the [documentation](https://www.scaleway.com/en/docs/managed-services/transactional-email/api-cli/webhook-events-payloads/) for full details.
```

> The `email_error` field has been replaced by two new fields: `response_code` and `response_message`. This change allows for more precise error identification and improved tracking of response details. See the [documentation](https://www.scaleway.com/en/docs/managed-services/transactional-email/api-cli/webhook-events-payloads/) for full details.

Reviewer 2: ```suggestion
The `email_error` field has been replaced by two new fields: `response_code` and `response_message`. This change allows for more precise error identification and improved tracking of response details. See the [documentation](/managed-services/transactional-email/api-cli/webhook-events-payloads/) for full details.
```
